tests/e2e/cucumber/publicLinkToFolderJourney.feature

```diff
@@ -31,3 +32,11 @@ Feature: Create public link to folder
     When the public reloads the public link pages
     Then the public should not see the following files on the files-drop page
       | textfile.txt |
+    When "Alice" opens the "files" app
+    # Archiver download is not implemented in OC10 yet
+    # https://github.com/owncloud/web/issues/6239 
+    # so this step is skipped in OC10
+    And "Alice" downloads the following files using batch action
+      | resource     | from         |
+      | lorem.txt    | folderPublic |
+      | textfile.txt | folderPublic |
```

Comment:
how about putting comment
```suggestion
      | textfile.txt | folderPublic |
    # Then what do we check for to be confident that the above things done by Brian have worked?
    # Then the downloaded files should have content "abc..."
```

Reply:
The downloaded file has the name `download.tar` and the assertion is done here https://github.com/owncloud/web/blob/6617eae4dd916d38004256ba7c3a00a8ebb0031e/tests/e2e/cucumber/step_definitions/app.files.ts#L200.
I don't know what do you mean by that. You want me to implement 
`Then the downloaded files should have content "ABC..." ` ?

Reply:
I think we implement  expect(download.suggestedFilename()).toBe('download.tar')  in then step

Reply:
In e2e we are asserting in the same step and not using a lot of then step unless absolutely necessary. Look here
https://github.com/owncloud/web/blob/0984f76d430902ec5011b135b17ae0dabc9f0092/tests/e2e/cucumber/steps/app-files/resource.ts#L58-L60